.pre-commit-config.yaml

```
@@ -31,3 +31,12 @@ repos:
     hooks:
       - id: pyupgrade
         args: [--py39-plus]
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
+  - repo: https://github.com/pre-commit/mirrors-clang-format
+    rev: v14.0.6
```

We currently have 13.0.1 pinned
```suggestion
    rev: v13.0.1
```